src/components/screens/EventRecord.js

```diff
@@ -182,6 +184,22 @@ export const EventRecord = ({ data, route }) => {
           title={texts.eventRecord.operatingCompany}
         />
 
+        {settings?.displayOnlySummary === 'true' && !!settings?.onlySummaryLinkText && (
```

Comment:
Is it possible to replace "true" with another variable? Why is the string value coming?

Reply:
i don't understand what you mean with replacing true?
`displayOnlySummary` comes as `'true'` or `'false'` per GraphQL and i don't know why it is not boolean.

Reply:
Okay, I got my answer. Only `'true'` or `'false'` string values are coming. 👍